behave_django/environment.py

```diff
@@ -1,3 +1,5 @@
+import copy
```

Comment:
Do a `from copy import copy` here, please. Makes the code easier to read in line 76.